python/src/trezorlib/cli/firmware.py

```diff
@@ -58,6 +81,24 @@ def _get_file_name_from_url(url: str) -> str:
     return os.path.basename(full_path)
 
 
+def _print_firmware_model(hw_model: Union[bytes, fw_models.Model]) -> None:
+    try:
+        model_name = fw_models.Model.from_hw_model(hw_model).name
+        click.echo(f"{model_name} firmware image.")
+    except ValueError:
+        pass
+    else:
```

Comment:
Could be more readable to `return` after `click.echo` 3 lines above - the same as is done 5 lines below

Reply:
40ac5672d33e78ffc2330f3abe2b6989a855d1df

i thought it's more readable with the else, but i admit you're right; try-else is rather uncommon construction